recipes/newrelic/apm/java/java-install.yml

```diff
@@ -0,0 +1,27 @@
+# Visit our schema definition for additional information on this file format
+# https://github.com/newrelic/open-install-library/blob/main/docs/recipe-spec/recipe-spec.md#schema-definition
+
+name: java-agent-installer
+displayName: Java Agent Installer
+description: New Relic install recipe for instrumenting Java applications
+repository: https://github.com/newrelic/newrelic-java-agent
+
+installTargets:
+  - type: application
+    os: linux
+  - type: application
```

Comment:
I think rishi said we would focus on java/linux only for now. If we do need windows, it's most likely going to be a different recipe file (powershell versus linux/bash)

Reply:
Split them out